sensor_msgs/srv/SnapshotCloud.srv

```diff
@@ -0,0 +1,17 @@
+# Point Cloud as a service
+
+# REQUEST:
+
+# dense_cloud = true, if the cloud has to be dense (ordered)
+bool dense_cloud
+
+# exposure in milliseconds (0 indicates autoexposure) 
+uint32 exposure
```

Comment:
Do you really expect to be setting the exposure time for a cloud? This seems to be breaking several abstractions that you must know about the sensor's parameters and lighting situation to request a snapshot?

Reply:
Do you mean to let the exposure config through another mechanism like, for instance, dynamic reconfigure, so keep the request even more simple ?

Reply:
Yes. This standalone argument means nothing unless you know a lot more about the system. It has implications for side effects too, such as will this modify the system if it's already running with the exposure setting? What does the exposure setting mean if this snapshot is implemented with a data source such as a laser?